frontend/src/app/app-routing.module.ts

```diff
@@ -1,9 +1,19 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { TokenExistsGuard } from './guards/token-exists.guard';
+import { TokenDoesnotExistGuard } from './guards/token-doesnot-exist.guard';
 
 const routes: Routes = [
-  { path: 'me', loadChildren: () => import('./modules/workspace/workspace.module').then(m => m.WorkspaceModule) },
-  { path: '', loadChildren: () => import('./modules/auth/auth.module').then(m => m.AuthModule) }
+  {
+    path: 'me',
```

Comment:
Change it to personal

Reply:
I think my version will be better, because then the child personal.component  will need a new route name like personalLibrary, and route will be xxxx/personal/personalLibrary, but previusly it looks like much better me/personal

Reply:
Since you have a guard, can you remove this path completely? User'll go to `.com/personal` like before in case he's logged in, and to `.com/login` or `.com/register` otherwise